src/promptflow/promptflow/_sdk/_serving/extension/extension_type.py

```diff
@@ -0,0 +1,12 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from enum import Enum
+
+
+class ExtensionType(Enum):
+    """Extension type used to identify which extension to load in serving app."""
+
+    Default = "local"
```

Comment:
Should enum use capitalized names? `Default` -> `DEFAULT`

Reply:
good suggestion, updated